sdks/nodejs/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google-cloud/agones-sdk",
-  "version": "1.3.0",
```

Comment:
@markmandel We may need to add a step to the process for publishing to prevent this file getting modified later as seen here and in other reviews. Here's what we do internally:
1. Increment version in package.json
2. Run `npm i` which will also update package-lock.json
3. Run `npm publish`

Reply:
I think we should revert the changes to this file in this PR and update nodejs dependencies independently of client-go dependencies.